cluster-scope/base/operators.coreos.com/subscriptions/cluster-logging/subscription.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: cluster-logging
   namespace: openshift-logging
 spec:
-  channel: stable-6.0
+  channel: stable-6.1
```

Comment:
It looks like the cluster-logging operator is installed on most of our clusters. Changing the version here will upgrade the cluster logging operator *everywhere*. Is that intentional? That already happened in d079066d9e6dbd24e1cba06a46f03156ca2fe71c with the change from 5.8 to 6.0.